docs/navigation-parameters-reference.md

```diff
@@ -16,29 +16,72 @@ Luigi.setConfig({
   // navigation structure and settings
   navigation: {
     nodeAccessibilityResolver: function (nodeToCheckPermissionFor, parentNode, currentContext) {},
-    nodes: [
-        // STATIC navigation node
-      {
-        pathSegment: 'settings',
-        label: 'Settings',
-        viewUrl: 'https://admin.mydomain.com/settings',
-        viewGroup: 'settingsGroup',
-        // optional
-        children: [node, node, node],
-        hideFromNav: false,
-        isolateView: false,
-        icon: 'settings'
+  viewGroupSettings: {
+    main: {
+      preloadUrl: 'https://my-site.com/index.html#/overview',
```

Comment:
The example preloadUrls should not lead to the impression that you should use any of your usual views. The preload URL view should be small, visually empty but must have LuigiClient included and must be fine with getting an empty context (e.g. without a token). Maybe this should be mentioned below more clearly.

Reply:
You are right, I added that info